typescript/web/freshpaint.ts

```diff
@@ -0,0 +1,38 @@
+// Install the freshpaint SDK: https://documentation.freshpaint.io/readme/guides/quickstart/installing-freshpaint
+
+// Import the Avo Codegen file from the location you've chosen
+import { CustomDestination } from "../path/to/Avo";
```

Comment:
```suggestion
import { FreshpaintDestinationInterface } from "../path/to/Avo";
```

Reply:
I believe that the Avo Codegen just spits out a single interface for all the destination interfaces called `CustomDestination`. Does that make sense?

Reply:
It would be cool to rename it to `Destination` at one point!